### PR TITLE
fix "please adopt containerbackground api" in widget after ios 17.0;

### PR DESCRIPTION
**Pull request by ynrng**
widget shows an error on initialisation after upgrading to iOS 17. This fix is based on https://stackoverflow.com/a/76842922/8262079. Tested on iOS 17.3.1 on iPhone.

### CODE REVIEW

File: GitHubContributionsWidget/GitHubContributionsGraphEntryView.swift [hunk 1]
```diff
@@ -34,6 +34,21 @@ struct GitHubContributionsGraphEntryView: View {
 
 // MARK: -
 
+
```
Comment (AnderGoig) [inline GitHubContributionsWidget/GitHubContributionsGraphEntryView.swift:37]:
```suggestion
```

File: GitHubContributionsWidget/GitHubContributionsGraphEntryView.swift [hunk 2]
```diff
@@ -34,6 +34,21 @@ struct GitHubContributionsGraphEntryView: View {
...
+            return containerBackground(for: .widget) {
+                backgroundView
+            }
+        } else {
+            return background(backgroundView)
+        }
+    }
+}
+
+
```
Comment (AnderGoig) [inline GitHubContributionsWidget/GitHubContributionsGraphEntryView.swift:51]:
```suggestion

```

**Review by AnderGoig — CHANGES_REQUESTED** [review]
thanks @ynrng, nice contribution!

can we a few code style changes? 🙏

**Comment by ynrng** [comment]
committed.

**Pull request merged**